examples/register-client-post-400.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Did you intend to include this file in this PR?

Reply:
it's being included in the other PR so it would be removed once that's merged in but I can remove it